package.json

```diff
@@ -40,7 +40,7 @@
     "flow-redux": "flow-coverage-report  -i 'src/reducers/*.js' -i 'src/actions/*.js'  -t text",
     "flow-react": "flow-coverage-report -i 'src/components/**/*.js' -t text",
     "storybook": "start-storybook -p 9001 -c .storybook",
-    "snapshot": "NODE_ENV='development' build-storybook && percy-storybook",
+    "snapshot": "NODE_ENV='development' build-storybook && percy-storybook --ignore=\"node_modules/(?!devtools-)\"",
```

Comment:
this was related to the babel plugin, which now transpiles devtools-reps which had a trailing comma :)